client/src/components/CommentApp/main.scss

```diff
@@ -1,56 +1,15 @@
-$color-teal: #007d7e;
-$color-teal-darker: darken(adjust-hue($color-teal, 1), 4);
-$color-teal-dark: darken(adjust-hue($color-teal, 1), 7);
+@import '../../../scss/settings/variables';
 
-$color-blue: #71b2d4;
-$color-red: #cd3238;
-$color-red-dark: #b4191f;
 $color-red-very-dark: #901419;
```

Comment:
question: should these make it into variables.scss and make use of CSS vars as the other `$color-` ones?

Reply:
ah, looking at https://github.com/wagtail/wagtail/blob/bd091b3784171da59cd30c3c155860c0eaa5525f/client/scss/settings/_variables.scss -- not all use css vars

Reply:
Yep, there's definitely a second phase of work needed to make the commenting app a "good citizen" regarding colour customisations - rather than just existing alongside them without breaking - but I think that requires insights into the front-end code that I don't have, for example knowing when these definitions belong in the main _variables.css, and whether they should have their own CSS vars or be derived from others with `darken()` etc.

Reply:
Regarding being a "good citizen", I think this is true with much of Wagtail's CSS - we have comparatively few CSS variables in general, almost all non-teals aren't able to be modified. This feels like an improvement to be made across the admin, to be honest.